src/user/auth/login-form/login-form.scss

```diff
@@ -32,9 +32,7 @@
 }
 
 .loginbtn {
-  border: 1px solid rgb(211, 220, 228);
-  // border-radius: 3px;
-  width: 45%;
+  border-radius: 100px !important;
```

Comment:
Using ! important, however, is bad practice and should be avoided because it makes debugging more difficult by breaking the natural cascading in your stylesheets. When two conflicting declarations with the ! important rule are applied to the same element.
Please chanage this.

Reply:
Resolved this!